### PR TITLE
Updating CPP and CS winui instancing samples.

### PR DESCRIPTION
**Pull request by dhoehna**
## Description
The winui samples were out of date and the instancing samples demonstrated more features than just instancing making the sample confusing to follow.

I made this change because I recently was on the WinUI show and promised samples.

## Target Release

1.0

## Checklist

My computer can't run ARM64.

- [ X] Samples build and run using the Visual Studio versions listed in the [Windows development docs](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio).
- [X ] Samples build and run on all supported platforms (x64, x86, ARM64) and configurations (Debug, Release).
- [X ] Samples set the minimum supported OS version to Windows 10 version 1809.
- [ X] Samples build clean with no warnings or errors.
- [ ] **[For new samples]**: Samples have completed the [sample guidelines checklist](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#checklist) and follow [standardization/naming guidelines](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#standardization-and-naming).
- [ ] If I am onboarding a new feature, then I must have correctly setup a new CI pipeline for my feature with the correct triggers and path filters laid out in the "Onboarding Samples CI Pipeline for new feature" section in samples-guidelines.md.
- [ ] I have commented on my PR `/azp run SamplesCI-<FeatureName>` to have the CI build run on my branch for each of my FeatureName my PR is modifying. This must be done on the latest commit on the PR before merging to ensure the build is up to date and accurate. Warning: the PR will not block automatically if this is not run due to '/azp run' limitation on triggering more than 10 pipelines.

### CODE REVIEW

File: Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing (Package)/Package.appxmanifest [hunk 1]
```diff
@@ -7,13 +7,13 @@
   IgnorableNamespaces="uap rescap">
 
   <Identity
-    Name="c4a3b17f-0039-493a-a7b5-a82222ffe041"
-    Publisher="CN=Microsoft Corporation"
+    Name="316a7f97-7cd3-4657-87b2-fd866c21e546"
+    Publisher="CN=dahoehna"
```
Comment (AdamBraden) [inline Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing (Package)/Package.appxmanifest:11]: I don't think you want your email alias in the repo. @btueffers what is the guidance here? should all samples have CN=Microsoft or something like CN=WinAppSDKSamples?

File: Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing/App.xaml [hunk 2]
```diff
@@ -1,6 +1,4 @@
-﻿<!-- Copyright (c) Microsoft Corporation.
- Licensed under the MIT License. -->
```
Comment (AdamBraden) [inline Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing/App.xaml:2]: why was the license removed?

File: Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/ActivationTrackerHelper.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            return Registry.CurrentUser.CreateSubKey(c_redirectionKeyName, true);
+        }
+
+        private static void SetNumberOfActivations(int newNumberOfActivations)
+        {
+            RegistryKey instanceTrackerKey = GetRegistryKey();
+            instanceTrackerKey.SetValue(c_redirectionValueName, newNumberOfActivations);
+        }
+
+        internal static int IncrementAndSetNumberOfRedirectionsReturnNewNumber()
```
Comment (AdamBraden) [inline Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/ActivationTrackerHelper.cs:26]: the name is a little odd - "IncrementAndSet*" and "*ReturnNewNumber". I'm not familiar with this level of detail in our api guidance. Given its internal/private, maybe not too bad?

File: Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing (Package)/Package.appxmanifest (hunk 1 above)
Comment (dhoehna) [reply Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing (Package)/Package.appxmanifest:11]: I think this file auto-updates when making a cert for publishing.  I did change it to Microsoft.

File: Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing/App.xaml (hunk 2 above)
Comment (dhoehna) [reply Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing/App.xaml:2]: No idea.  I added it back in.

File: Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/ActivationTrackerHelper.cs (hunk 3 above)
Comment (dhoehna) [reply Samples/AppLifecycle/Instancing/cs-winui-packaged/CsWinUiDesktopInstancing/ActivationTrackerHelper.cs:26]: Yeah.  It is a bit wordy.  I just wanted to be descriptive on what the method does including all side-effects.

But, as you said, because this is internal and not production ready code, I think the smaller name works here.

File: Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing/App.xaml (hunk 2 above)
Comment (dhoehna) [reply Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing/App.xaml:2]: The license was removed because it is failing compilation.   Xaml Internal Error error WMC9999: Data at the root level is invalid. Line 1, position 2.

Build is successful when the build is removed.  I'll try and see what is wrong.

Comment (dhoehna) [reply Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing/App.xaml:2]: Fixed the issue.  Somehow , one of those characters with 2 spaces in them was added to the start of the comment.
